### PR TITLE
Nataliia/feat add scroll spy

### PR DESCRIPTION
**Pull request by NataliiaKononikhina**

### CODE REVIEW

**Comment by kparzecki** [comment]
I think it would nice to have `cursor: pointer` attribute on `.landing-page__button`.

**Comment by kparzecki** [comment]
image
There is a typo in `Certified`.

File: user/themes/centra/scss/landing-page/_code-animation.scss [hunk 1]
```diff
@@ -0,0 +1,238 @@
...
+    white-space: nowrap;
+    overflow: hidden;
+  }
+}
+
+.hero-section {
+  .css-typing p:nth-child(1) {
+    width: 18em;
+    animation: type 4s steps(40, end);
+    animation-fill-mode: forwards;
```
Comment (kparzecki) [inline user/themes/centra/scss/landing-page/_code-animation.scss]: You don't have to repeat `animation-fill-mode` for every child. You can add this to `p` element.

**Comment by jagodapp** [comment]
Something happened to the logo here? 🤔 

Zrzut ekranu 2019-05-06 o 12 05 16

File: user/themes/centra/scss/landing-page/_code-animation.scss [hunk 2]
```diff
@@ -0,0 +1,228 @@
...
+
+.orange {
+  color: #FFD595;
+}
+
+.light-blue {
+  color: #D0D4F0;
+}
+
+.blue{
```
Comment (kparzecki) [inline user/themes/centra/scss/landing-page/_code-animation.scss]: Please add space before `{`

File: user/themes/centra/scss/landing-page/_code-animation.scss [hunk 3]
```diff
@@ -0,0 +1,228 @@
...
+
+.pink {
+  color: #FFD3DA;
+}
+
+.green {
+  color: #18AE1D;
+}
+
+.code-animation{
```
Comment (kparzecki) [inline user/themes/centra/scss/landing-page/_code-animation.scss]: And here

**Pull request merged**